src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, division
+
+import logging
+
+from django.utils import timezone
+
+from sentry.app import locks
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger('sentry.scheduler')
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    with locks.get('scheduler.process', duration=60).acquire():
+        queryset = ScheduledJob.objects.filter(
+            date_scheduled__lte=timezone.now(),
+        )
+        job_count = queryset.count()
+        if job_count > 100:
```

Comment:
Probably better to just select 101 and see if size of the size of the result exceeds 100 to avoid making the database do the query twice. I'm not sure the exact number matters, we just want to be able to figure out if we're lagging behind or not.